client/dashboard/data/emails.ts

```diff
@@ -1,4 +1,12 @@
-export type EmailProvider = 'titan' | 'google-workspace' | 'forwarding';
+import wpcom from 'calypso/lib/wp';
+export type EmailProvider = 'titan' | 'google_workspace' | 'forwarding';
```

Comment:
Should this be `email_forwarding`?

```suggestion
export type EmailProvider = 'titan' | 'google_workspace' | 'email_forwarding';
```

<img width="1496" height="592" alt="image" src="https://github.com/user-attachments/assets/c48642d5-f7c7-4506-a4cf-497a6e43a343" />

Reply:
Fixed. Thanks for reporting it. 👌